core/src/com/threecubed/auber/World.java

```diff
@@ -436,4 +443,35 @@ public void checkForEndState() {
       game.setScreen(new GameOverScreen(game, true));
     }
   }
+
+  public enum Abilities {
+    superspeed,
+    invisibility,
+    invincibility,
+    insta_beam,
+    vision;
```

Comment:
nit: conventionally these should be UpperCamelCase

Reply:
That can be changed, I'll have to change the texture file names first

Reply:
Done